### PR TITLE
:bug: unable to retrieve keypair is basically when someone clears localstorage

### LINKED ISSUE #3591: Error: Unable to retrieve keypair
### What happened?

Mint not working last 4 hours on RMRK network....

### Please reproduce in steps

classic NFT minting for the collection...

### Expected Behavior

classic NFT minting for the collection

### What browsers are you seeing the problem on?

Chrome, Firefox

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### At which chain did you encounter bug?

RMRK

### Screenshots

obrázok

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer?target=FqCJeGcPidYSsvvmT17fHVaYdE2nXMYgPsBn3CP9gugvZR5

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #7187: Clearing localstorage will glitch the app (Unable to retrieve a keypair)
Artist (tg: @Frankford18) is trying to mint his collection: https://kodadot.xyz/ahk/collection/120
Production, AssetHub

He is getting this error:

What it is?

### PR DESCRIPTION
**Pull request by vikiival**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [ ] Closes #3591
- [ ] Close #7187

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

File: utils/transactionExecutor.ts [hunk 1]
```diff
@@ -34,6 +34,11 @@ const exec = async (
     const transfer = await callback(...params)
     const address = typeof account === 'string' ? account : account.address
     const injector = await getAddress(toDefaultAddress(address))
+
+    if (!injector) {
+      throw new Error('Ups! We cannot find your wallet, please login again.')
```
Comment (roiLeo) [inline utils/transactionExecutor.ts:39]:
```suggestion
      throw new Error('Oops! We can't find your wallet, please log in again.')
```

File: utils/transactionExecutor.ts [hunk 2]
```diff
@@ -34,6 +34,11 @@ const exec = async (
     const transfer = await callback(...params)
     const address = typeof account === 'string' ? account : account.address
     const injector = await getAddress(toDefaultAddress(address))
+
+    if (!injector) {
+      throw new Error('Oops! We can't find your wallet, please log in again.')
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply utils/transactionExecutor.ts:39]: oh no my bad
